import-automation/executor/app/executor/cloud_run_simple_import.py

```diff
@@ -0,0 +1,217 @@
+    import_script: Specific import script being processed
+
+  Returns:
+    job_is as a string simple-import-<import_name> for simple-imports
+    or '' for other imports.
+  """
+    # Create a job is for simple import cloud run as
+    # <config_path>-<import_name>
+    import_name = import_spec.get('import_name')
+    import_pos = import_script.find(_IMPORT_CONFIG_DIR)
+    if import_pos < 0 or _DEFAULT_CONFIG_PREFIX not in import_script:
+        # Not a simple import
+        return ''
+    # Get the import config path under .../simple_imports/
+    job_str = [
+        import_script[import_script.find(_IMPORT_CONFIG_DIR) + 1:].replace(
+            _DEFAULT_CONFIG_PREFIX, '').removesuffix('.json')
+    ]
+    if import_name:
+        job_str.append(import_name)
+    return cloud_run.get_job_id('-'.join(job_str))
+
+
+def get_simple_import_gcs_config(
+    config_file: str,
+    import_name: str,
+    version: str,
+    gcs_bucket: str = _DEFAULT_GCS_BUCKET,
+    config_dir: str = _IMPORT_CONFIG_DIR,
+    copy_file: bool = True,
+) -> str:
+    """Returns path to a copy of the import config file on GCS.
+
+  Simple imports are processed as a Cloud Run job that cannot access local
+  files.  So import config and output files are saved in the GCS folder:
+    <GCS-Bucket>/simple_imports/<import>/<version>/
+
+  Args:
+    config_file: Import config json
+    import_name: name of the import
+    version: dated version string for the import run
+    gcs_bucket: GCS bucket for the import config and outputs.
+    copy_file: if True copies the config file to GCS.
+
+  Returns:
+    GCS path to the config file,
+    like gs://<GCS-BUCKET>/simple_imports/<import>/<version>/import_config.json
+  """
+    path = []
+    if config_dir.startswith('gs://'):
+        # Use the provided gcs folder
+        path.append(config_dir)
+    else:
+        # Use the gcs path: gs://<bucket>/simple_imports/<import>/
+        if not gcs_bucket:
+            gcs_bucket = _DEFAULT_GCS_BUCKET
+        path.append(gcs_bucket)
+        path.append(config_dir)
+        config_file_path = os.path.dirname(config_file)
+        config_pos = config_file_path.find(config_dir)
+        if config_pos >= 0:
+            config_pos += len(config_dir)
+        else:
+            config_pos = 0
+        path.extend(config_file_path[config_pos:].split('/'))
+        path.append(import_name)
+    path.append(version)
+    path.append(os.path.basename(config_file))
+    gcs_config_file = '/'.join([d.strip('/') for d in path if d])
+    if not gcs_config_file.startswith('gs://'):
+        gcs_config_file = 'gs://' + gcs_config_file
+    if copy_file and config_file != gcs_config_file:
+        logging.info(
+            f'Copying import config: {config_file} to {gcs_config_file}')
+        file_util.file_copy(config_file, gcs_config_file)
+    return gcs_config_file
+
+
+def cloud_run_simple_import_job(
+    import_spec: str,
```

Comment:
Can we add a TODO to create a spec programmatically rather than asking partners to provide one?

Reply:
having import automation use import_config.json instead of manifest.json will need more changes. Tracking in the doc. For now, I'm expecting partners to just create the import_config.json and the internal team will create the manifest.json along with other import manifests, validations, etc.

Reply:
Added a TODO